tests/test-war-smoke/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM ${jetty.test.image}
+ADD root /app
```

Comment:
I'm using `jetty:9.3` that is built by `mvn clean install` from the project root. I think the problem is that the files in my working directory have different permissions and those permissions are propogated through Docker's `ADD`, but I haven't verified that yet. I'll dig a bit more.